guide/daily-spending-wallet/receiving.md

```diff
@@ -0,0 +1,194 @@
+redirect_from:
+ - /guide/payments/receive/
+ - /guide/daily-spending-wallet/funding/
+ - /guide/onboarding/funding-a-wallet/
+main_classes: -no-top-padding
+---
+
+{% include picture.html
+    image = "/assets/images/guide/daily-spending-wallet/requesting/request-banner(1600x600).png"
+    retina = "/assets/images/guide/daily-spending-wallet/requesting/request-banner(3200x1200).png"
+    mobile = "/assets/images/guide/daily-spending-wallet/requesting/request-banner(400x200).png"
+    mobileRetina = "/assets/images/guide/daily-spending-wallet/requesting/request-banner(800x400).png"
+    alt-text = "Image: Text balloon containing a bitcoin."
+    width = 1600
+    height = 600
+    layout = "full-width"
+%}
+
+<!--
+
+Illustration source
+
+- https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/Design-Guide-Bosch?node-id=201%3A687
+
+-->
+
+# Receiving bitcoin
+
+Once a user has shared a payment request, the next step is receiving the payment. Ideally, there is little to no actions for a user to take at this stage of the payment flow. 
+
+There are however several things to consider as a product designer to make this step as frictionless as possible. For instance, users need to be online and have a payment channel with inbound liquidity to receive a payment. 
+
+This page covers how to design with these constraints in mind, as well as give an overview of how receiving a payment works.
+
+## Receiving offline
+
+Users can not receive lightning payments non-custodially offline. This is problematic for mobile users who are regularly offline. This means there will be times when a payment can not be received.
+
+With a [lightning wallet server]({{ '/guide/how-it-works/lightning-services/#receive-payments-offline' | relative_url }}) a payment can be intercepted and held for a user until they open their wallet. A push notification can be sent to the user so they can open their wallet and receive the payment. 
+
+Another option is to educate users to keep their wallet open. This can be explained during [first use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}) or when the user attempts to close their wallet.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/ReceiveOfflineNotification.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/ReceiveOfflineNotification@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/ReceiveOfflineNotification@2x.png"
+   layout = "shadow"
+   caption = "Notify the user that they have a payment incoming and that they should open their app to receive it."
+   alt-text = "Screen showing a notification that is asking the user to open their wallet so they can receive their payment."
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/CloseWalletNotification.png"
```

Comment:
This makes it sound like I can never close the app ever...? How about only showing this prompt right after the user has created an invoice until it has either been paid or some time of timeout?

Reply:
Good point, will include a note on this on the requesting page too

Reply:
Added to requesting page and removed notification modal image from this page.